### PR TITLE
Add Metering Operator to catalog

### PR DESCRIPTION
**Pull request by EmilyM1**

### CODE REVIEW

File: deploy/chart/catalog_resources/rh-operators/metering-operator.v0.12.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+    name: Red Hat
+  labels:
+    alm-owner-metering: metering-operator
+    alm-status-descriptors: metering-operator.v0.12.0
+  selector:
+    matchLabels:
+      alm-owner-metering: metering-operator
+  installModes:
+  - type: OwnNamespace
+    supported: false
```
Comment (chancez) [inline deploy/chart/catalog_resources/rh-operators/metering-operator.v0.12.0.clusterserviceversion.yaml]: This one should be true

File: manifests/0000_30_07-certified-operators.configmap.yaml [hunk 2]
```diff
@@ -1285,7 +1285,7 @@ data:
       
                     containers:
                     - name: mongodb-enterprise-operator
-                      image: quay.io/mongodb/mongodb-enterprise-operator:0.3
+                      image: registry.connect.redhat.com/mongodb/enterprise-operator
```
Comment (chancez) [inline manifests/0000_30_07-certified-operators.configmap.yaml]: This shouldn't be in your PR. Looks like you may have rebased incorrectly?

Comment (EmilyM1) [reply manifests/0000_30_07-certified-operators.configmap.yaml]: didn't rebase

File: deploy/chart/catalog_resources/rh-operators/metering-operator.v0.12.0.clusterserviceversion.yaml (hunk 1 above)
Comment (EmilyM1) [reply deploy/chart/catalog_resources/rh-operators/metering-operator.v0.12.0.clusterserviceversion.yaml]: Fixed

**Pull request merged**